### PR TITLE
fix: package manager field

### PR DESCRIPTION
**Pull request by kovsu**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

我使用的 [ni](https://github.com/antfu/ni) 进行依赖安装时，出现下面 unkown packageManager 这个情况

查看了 [官方文档](https://nodejs.org/api/packages.html#packagemanager)，`<package manager name>@<version>` 这种才是正确的命名规范

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     -      |
| 🇨🇳 Chinese |     -      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 7194169</samp>

Fixed a bug related to npm 7 compatibility by specifying `npm@9.0.0` as the `packageManager` in `package.json`. This ensures consistent and compatible package management across different environments.

### <samp>🤖 Generated by Copilot at 7194169</samp>

* Fix npm 7 compatibility issues by using a specific version of npm as the package manager ([link](https://github.com/ant-design/ant-design/pull/45327/files?diff=unified&w=0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L4-R4))

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "antd",
   "version": "5.10.0",
-  "packageManager": "^npm@9.0.0",
+  "packageManager": "npm@9.0.0",
```
Comment (yoyo837) [inline package.json:4]:
```suggestion
  "packageManager": "npm@9",
```

是不是这样更OK?

Comment (kovsu) [reply package.json:4]: 应该要给一个确切的版本号吧

**Comment by yoyo837** [comment]
Rebase to fix CI failed.

**Comment by kovsu** [comment]
@yoyo837 done, please review

**Review by afc163 — APPROVED** [review]

**Comment by Wxh16144** [comment]
额，我之前也是用的 ni ，但是我 fork 了 antfu/ni 做了自定义 , 也一直出现这个提示，然后就没管。今天发现被改回去了😂

Duplicate of #44300

**Comment by Dunqing** [comment]
ni 和 nodejs 文档都该更新了

**Comment by kovsu** [comment]
我是看到 nodejs 文档里面有一句 `ensure that your teams use the exact same package manager versions`

**Comment by Dunqing** [comment]
> 我是看到 nodejs 文档里面有一句 `ensure that your teams use the exact same package manager versions`

文档没问题，只是太久没更新。最新的实现你可以看 corepack 的代码，是支持范围的

**Comment by kovsu** [comment]
> 额，我之前也是用的 ni ，但是我 fork 了 antfu/ni 做了自定义 , 也一直出现这个提示，然后就没管。今天发现被改回去了😂
> 
> Duplicate of #44300

我的问题，没看到你这个 pr 😂

**Pull request merged**